packages/devtools-proxy-support/src/system-ca.ts

```diff
@@ -89,23 +78,84 @@ export function removeCertificatesWithoutIssuer(ca: string[]): {
     }
     return { pem, parsed };
   });
-  caWithParsedCerts = caWithParsedCerts.filter(({ parsed }) => {
-    const keep =
-      !parsed ||
-      parsed.checkIssued(parsed) ||
-      caWithParsedCerts.find(
-        ({ parsed: issuer }) => issuer && parsed.checkIssued(issuer)
-      );
-    if (!keep) {
-      messages.push(
+}
+
+function doesCertificateHasMatchingIssuer(
```

Comment:
grammar nit: have, not has :)

Reply:
TY! Will fix

Reply:
so either doesCertificateHaveMatchingIssuer or certificateHasMatchingIssuer